### DISCUSSION TITLE
Unable to integrate Django, React and vite.

### DISCUSSION THREAD

**Discussion by Waqar-SE** [MrBin99/django-vite#135] [Q&A] [answered]
A full description of issue is below. 
https://stackoverflow.com/questions/78554185/unable-to-integrate-django-vite

Essentially I found some solutions but those are outdated, not sure what I'm doing wrong.

**Comment by Waqar-SE** [comment]
<img width="1094" alt="afbeelding" src="https://github.com/MrBin99/django-vite/assets/36355455/a96fcbf5-3c33-4e20-8367-e5d3d958a5c9">

**Reply by Waqar-SE** [reply]
@MrBin99 any suggestions ??

**Reply by MrBin99** [reply, marked as answer]
Sorry I'm not a React developer pour I think your are missing this `{% vite_react_refresh %}`.
Check out the documentation.

**Reply by Waqar-SE** [reply]
Thanks, that was indeed what I was missing.
